src/services/node-bridge-data.provider.ts

```diff
@@ -2,47 +2,55 @@ import {inject} from '@loopback/core';
 import {getLogger, Logger} from 'log4js';
 import {ServicesBindings} from '../dependency-injection-bindings';
 import {BridgeDataFilterModel} from '../models/bridge-data-filter.model';
-import {Log} from '../models/rsk/log.model';
 import {RskBlock} from '../models/rsk/rsk-block.model';
-import {RskNodeService} from './rsk-node.service';
 import FilteredBridgeTransactionProcessor from './filtered-bridge-transaction-processor';
 import RskBlockProcessorPublisher from './rsk-block-processor-publisher';
 import {bridge} from '@rsksmart/rsk-precompiled-abis';
+import {Transaction} from 'bridge-transaction-parser';
+import {BridgeService} from './bridge.service';
+
+export interface ExtendedBridgeTx extends Transaction {
+  blockHash: string;
+  createdOn: Date;
+  to: string | null;
```

Comment:
why would `to` be null?

Reply:
Because the type of web3Tx.to, for some reason, is also `to: string | null;`. I could change it and cast it to string if we know it will never be null.